README.md

```diff
@@ -87,13 +87,13 @@ configuration options [here](docs/options.md). If you are on Windows, check [her
 Install the IronFunctions CLI tool:
 
 ```sh
-curl -sSL http://get.iron.io/fnctl | sh
+curl -sSL http://get.iron.io/fn | sh
```

Comment:
Mapped it to: https://raw.githubusercontent.com/iron-io/functions/master/fn/install.sh

That correct?